sources/packages/web/src/views/student/StudentApplicationSummary.vue

```diff
@@ -7,7 +7,8 @@
         </template>
       </header-navigator>
     </template>
-    <v-row>
+    <v-row
+      >student-applications
```

Comment:
I believe that the string `student-applications` is lost in the code.

Reply:
oh..thanks for that catch, I will remove it.